pallets/anchors/src/common.rs

```diff
@@ -1,11 +1,29 @@
+// Copyright 2021 Centrifuge Foundation (centrifuge.io).
+// This file is part of Centrifuge chain project.
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
 use frame_support::storage::child::ChildInfo;
 
-pub const MS_PER_DAY: u64 = 86400000;
+// FIXME (ToZ):
+// Will be better to use runtime-common constant instead. However
+// there is a circular dependeny between runtime-common and this
+// create :( (due to AnchorData struct). Perhaps a good idea to
+// move AnchorStruct to runtime-common impl module.
```

Comment:
Might be better to move `MILLISECS_PER_DAY` to a `primitives` crate, so that we have a clear primitives -> pallets -> runtime dependency chain?

Reply:
Hi Branan. I fully agree. It would be nice to have even `primitives/libs->pallets->runtime(s)->node(s)`. Let's decide during our next protocol-dev-sync session next Tuesday (October 5th). Ciao Branan